AUTHORS.md

```diff
@@ -14,6 +14,7 @@ SMT has been developed thanks to contributions from:
 * Alexandre Thouvenot
 * Andres Lopez Lopera
 * Antoine Averland
+* Enrico Stragiotti
```

Comment:
nitpicking: you should be listed just after Emile  😉

Reply:
Oups, my bad 😅